verification/sessionmanager/sessionmanager_ttlcache.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2022 Contributors to the Veraison project.
```

Comment:
Just a question on file naming.

As the file is in sessionmanager, can we change the file to 

just ttlcache.go